src/modules/translation/languages/en.json

```diff
@@ -71,6 +71,7 @@
   "topbar": {
     "add_to_contest": "Add to contest",
     "add_to_contest_disabled_limits": "Too many {metric}{br}Try deleting some items!",
+    "add_to_contest_disabled_bounds": "Item out of boundaries{br}Try to move it in!",
```

Comment:
We might need to change this copy, not sure if `"move it in!"` is entirely correct @twaldorf

Reply:
```suggestion
    "add_to_contest_disabled_bounds": "Item out of bounds{br}Move the item inside the boundaries!",
```

Is that too long?

Reply:
Is this the only thing blocking?

Reply:
done ✅